frontend/src/routes/createLearningKit.svelte

```diff
@@ -0,0 +1,115 @@
+<script lang="ts">
+  import '../styles/lernello-theme.css';
+  import SuperDebug, { superForm } from 'sveltekit-superforms';
+
+  let { data } = $props();
+  const { form, errors, enhance } = superForm(data.form);
+</script>
+
+<form
+        method="POST"
+        use:enhance
+        action="?/CreateLearningKit"
+>
+    <h1>Create Learning Kit</h1>
+    <div>
+      <label for="name">Name*:</label>
```

Comment:
Make use uf skeleton ui, heres the docs for forms i.e.
https://www.skeleton.dev/docs/tailwind/forms